rpm_spec/core-agent.spec.in

```diff
@@ -310,12 +324,6 @@ fi
 
 %triggerin -- grub2-tools
 
-if ! grep -q /etc/default/grub.qubes /etc/default/grub 2>/dev/null; then
-    # do not keep Qubes-related settings directly in user-controlled config,
-    # include another file
-    echo '. /etc/default/grub.qubes' >> /etc/default/grub
-fi
```

Comment:
Where did this go? From what I know, Fedora don't support `/etc/default/grub.d` yet, so sadly we still need this hack.

Reply:
I have restored it.